django_comments/compat.py

```diff
@@ -0,0 +1,6 @@
+import django
+
+if django.VERSION < (1, 7):
+    from django.contrib.sites.models import get_current_site
+else:
+    from django.contrib.sites.shortcuts import get_current_site
```

Comment:
`from django.contrib.sites.shortcuts import get_current_site` will raise an `ImportError` on old versions of django. We can fallback to `sites.models` if we get an ImportError.